.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+
+name: Python package
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Comment:
doesn't even need a matrix... can just drop the `strategy` and use 3.8 (or 3.9) in the `python-version`